hazelcast/src/test/java/com/hazelcast/client/impl/spi/impl/ClientInvocationServiceImplTest.java

```diff
@@ -70,6 +82,12 @@ private ClientConnection closedConnection() {
         return conn;
     }
 
+    private ClientConnection mockConnection() {
+        ClientConnection connection = mock(ClientConnection.class, RETURNS_DEEP_STUBS);
```

Comment:
As I learned from documentation, RETURNS_DEEP_STUBS is useful when mock a mocked method should return a mock object. (nested mock object). And I learned that `RETURNS_DEEP_STUBS` is not a recommended feature in a clean code. Also I think we don't need it as the tests pass without it and there is not nested mock objects.

So can we remove it from this method and the method above?

Reply:
Honestly, I copy pasted it from that method. Let me remove both